From f842629226aa6a1014b6152f9be57ee5e4528843 Mon Sep 17 00:00:00 2001
From: Ran Shalit <rshalit@habana.ai>
Date: Mon, 30 Nov 2020 15:49:41 +0200
Subject: [PATCH] [SW-0] goya: ppboot: fix check of valid preboot

Fix wrong return value, which caused a preboot
checksum failure in goya.

Change-Id: I36c981aaa7d7d3475d6511d38eacf01a24cc9606
Signed-off-by: Ran Shalit <rshalit@habana.ai>
---

diff --git a/ppboot/ppboot.c b/ppboot/ppboot.c
index b0bdd2c..d2a011d 100644
--- a/ppboot/ppboot.c
+++ b/ppboot/ppboot.c
@@ -22,7 +22,7 @@
 
 __weak bool ppboot_is_valid_img_desc(struct boot_desc *desc)
 {
-	return 0;
+	return true;
 }
 
 static bool ppboot_try_candidate_img(enum fw_post_reset_cmd image_pending,
